openshift-ci/Dockerfile.tools

```diff
@@ -17,4 +17,14 @@ RUN mkdir -p $GOPATH/src/github.com/operator-framework \
 # install jq
 RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.6/jq-linux64 \
     && chmod +x ./jq \
-    && cp jq /usr/bin
\ No newline at end of file
+    && cp jq /usr/bin
+
+# install nodejs
+ENV NODE_VERSION=12.16.3 \
+    NPM_CONFIG_CACHE=/tmp/.npm
+
+RUN curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-x64.tar.xz" \
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
+    && tar -xJf "node-v$NODE_VERSION-linux-x64.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
+    && rm "node-v$NODE_VERSION-linux-x64.tar.xz" \
+    && ln -s /usr/local/bin/node /usr/local/bin/nodejs 
```

Comment:
This is going to conflict with the changes made here https://github.com/integr8ly/integreatly-operator/pull/725/files#diff-48abca7cd218705cd020054b38fa29cd. Those should be merged back soon, will need to update this after it merges before we can merge this.